lua/orgmode/files/file.lua

```diff
@@ -269,6 +269,21 @@ function OrgFile:find_headline_by_title(title)
   end)
 end
 
+function OrgFile:get_todo_keywords()
+  local todo_directive = self:_get_directive('todo')
+  if not todo_directive then
+    return config:get_todo_keywords()
+  end
+
+  local keywords = vim.split(todo_directive, '%s+')
```

Comment:
Do we need to trim `todo_directive` to remove excess spaces?
```suggestion
  local keywords = vim.split(vim.trim(todo_directive), '%s+')
```